<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30px;
border:2px  solid #00007f;
padding:10px;
background-color: qlineargradient(spread:pad, x1:0.114697, y1:0.103, x2:1, y2:1, stop:0.447761 rgba(5, 70, 207, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>501</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,0,0,0%);
border:none;
background-color: transparent;
border-radios:40px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Downloads/centro-recreacional.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEnviar">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 194, 126);
border-color: rgb(192, 191, 188);
border-color: rgb(222, 221, 218);
color: rgb(0, 0, 0);
font: 75 12pt &quot;Arial&quot;;
border-radius:15px;

QPushButton{
	background-color: rgb(87, 227, 137);

}
QPushButton:hover{
	background-color: rgb(143, 240, 164);

}</string>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHistorial">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 211, 45);
color: rgb(0, 0, 0);
border-color: rgb(222, 221, 218);
font: 75 10pt &quot;Arial&quot;;
border-radius:15px;

QPushButton{
	background-color: rgb(224, 27, 36);

}
QPushButton:hover{
	background-color: rgb(237, 51, 59);

}</string>
    </property>
    <property name="text">
     <string>Historial</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameChatBot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>561</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(246, 211, 45);
background-color: rgb(222, 221, 218);
border-color: rgb(248, 228, 92);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frameChat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>541</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(153, 193, 241);
border-radius:10px;
border:none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(36, 31, 49);</string>
      </property>
      <property name="text">
       <string>Chat:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelChatTitle">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
border:none;</string>
      </property>
      <property name="text">
       <string>Hola, soy tu chatbot de actividades de recreación. ¿Que deseas actividad deseas?</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frameUser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>541</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(143, 240, 164);
border-radius:10px;
border:none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(36, 31, 49);</string>
      </property>
      <property name="text">
       <string>Tu:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelUser">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frameChat_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>541</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(153, 193, 241);
border-radius:10px;
border:none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(36, 31, 49);</string>
      </property>
      <property name="text">
       <string>Chat:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelChat">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="txtEnviar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px;
background-color: rgb(255, 255, 255);
border-color: rgb(143, 240, 164);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 27, 36);
color: rgb(0, 0, 0);
border-color: rgb(222, 221, 218);
font: 75 10pt &quot;Arial&quot;;
border-radius:15px;

QPushButton{
	background-color: rgb(224, 27, 36);

}
QPushButton:hover{
	background-color: rgb(237, 51, 59);

}</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNombreUsuario">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>20</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 209, 122);
background-color: transparent;
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
